code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -120,3 +120,11 @@
 	volume = 60
 	amount_per_transfer_from_this = 30
 	list_reagents = list("atropine" = 10, "epinephrine" = 10, "salbutamol" = 20, "spaceacillin" = 20)
+
+/obj/item/weapon/reagent_containers/hypospray/medipen/survival

```

Comment:
Our conversation vanished but

1) I agree this is obviously more efficient than asking chemistry for stuff

2) I'm worried though because "I can't rely on anyone else, it takes too much time" is what led to every job being a self sufficient R-walled box where everyone is bored to death